### PR TITLE
Porting maui combobox test to xUnit

### PR DESCRIPTION
**Pull request by AdamYoblick**

### CODE REVIEW

**Comment by AdamYoblick** [comment]
@zsd4yr @RussKie please take a look when you get a chance. This is me porting one more maui test and also fixing issues that only showed up when running multiple tests. I recording the porting of this test for the test team and will put the video on our teams sharepoint.

File: src/System.Windows.Forms/tests/IntegrationTests/MauiTests/Directory.Build.props [hunk 1]
```diff
@@ -2,11 +2,6 @@
 
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
```
Comment (zsd4yr) [inline src/System.Windows.Forms/tests/IntegrationTests/MauiTests/Directory.Build.props]: Why was this moved out?

Comment (AdamYoblick) [reply src/System.Windows.Forms/tests/IntegrationTests/MauiTests/Directory.Build.props]: It was moved because visual studio errors out when loading a csproj that doesn't contain some of this stuff. So I couldn't make it all common unfortunately. Ran into that when I tried adding a second test. They have to copy paste the existing csproj anyway so I was ok with it.

Comment (zsd4yr) [reply src/System.Windows.Forms/tests/IntegrationTests/MauiTests/Directory.Build.props]: That is absurd! Who can I yell at about this?!?!?!
